api/xml/elife/articles/00569.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" ?>
+<?xml-model href="../article.rng"?>
```

Comment:
If this was returned by the API the `xml-model` reference should go a URL, which is in the specification.